### ISSUE TITLE
Quantum Multiverse continually showing it needs to be updated in VCV Rack 2.1.1

### ISSUE THREAD

**Issue by pgatt** [martin-lueders/ML_Pro_Series#10]
Hi Martin,

Due to a separate issue that we've fixed, Quantum Multiverse is now incorrectly showing that it needs to be updated.

Could you please remove lines of code that overwrite p->slug and p->version and then request an update?

We appreciate your time.

**Comment by rubyglow** [comment]
Yeah, that'd be great Martin. The update-loop is slightly annoying.

**Comment by rubyglow** [comment]
Oh, never mind. I see you fixed it with the latest update today. Thanks!
